docs/model-hub/transformers/index.txt

```diff
@@ -0,0 +1,82 @@
+.. _model-hub-transformers:
+
+########################
+ Model Hub Transformers
+########################
+
+.. rubric:: **Overview**
+
+`The Huggingface transformers library
+<https://github.com/huggingface/transformers>`_ is the de-facto library
+for natural language processing models. It provides pretrained weights
+for leading NLP models and allows users to easily use these pretrained
+models for the most common NLP tasks like language modeling, text
+classification, and question answering among others.
+
+**model-hub** makes it easy for users to train transformer models in
+Determined while keeping the developer experience as close as possible
+to what it's like working directly with **transformers**. Our library
+serves as an alternative to HuggingFace's `Trainer Class
+<https://huggingface.co/transformers/main_classes/trainer.html>`_ and
+provides access all of Determined's benefits including:
+
+-  Easy :ref:`multi-node distributed training <multi-gpu-training>` with
+   no code modifications. Determined automatically sets up the
+   distributed backend for you.
+
+-  Experiment tracking, experiment visualization, artifact tracking, and
```

Comment:
Experiment visualization as a thing doesn't make too much sense to me. What specifically are you visualizing here -- the loss accuracy curves or the hyperparameters? Dunno if I'm being too nitpicky on this so feel free to ignore and resolve this comment if I am.